core/src/lib/lex.cc

```diff
@@ -765,6 +765,10 @@ int LexGetToken(LEX* lf, int expect)
           break;
         }
         if (ch == L_EOL) {
+#if HAVE_WIN32
+          add_str(lf, '\r');
+#endif
```

Comment:
Should we add an `\r` here or not ? Im not sure what the best approach is.

Reply:
Makes sense.  I figured that there might be windows specific things that may need windows line endings to work. But i guess in that case we probably wouldve gotten bug reports by now as that is currently also not possible.